samples/storage_csm_powerflex_v240.yaml

```diff
@@ -94,13 +92,6 @@ spec:
 
     node:
       envs:
-        # X_CSI_HEALTH_MONITOR_ENABLED: Enable/Disable health monitor of CSI volumes from node plugin - volume usage
-        # Allowed values:
-        #   true: enable checking of health condition of CSI volumes
-        #   false: disable checking of health condition of CSI volumes
-        # Default value: false
-        - name: X_CSI_HEALTH_MONITOR_ENABLED
```

Comment:
Hi @jooseppi-luna @bharathsreekanth , If https://github.com/dell/csm/issues/713 was all about the config map, why  X_CSI_HEALTH_MONITOR_ENABLED env var was removed as a part of this PR?